gravityflow/gravityflowformconnector
# Add support for REST API v2

## Description

This updates the steps to offer REST API v2 as an option for interacting with remote forms/entries.

## Testing Instructions

- Use the update-entry-filter Gravity Flow branch
- Test that the remote site actions of existing New Entry, Update an Entry, Update Fields, and Delete an Entry steps continue to function using REST API v1.
- Configure the four steps to use REST API v2 and confirm the actions are performed correctly
